ompi/mca/osc/ucx/osc_ucx_comm.c

```diff
@@ -268,11 +268,12 @@ static inline int get_dynamic_win_info(uint64_t remote_addr, ompi_osc_ucx_module
          * Therefore, force lock is needed. Remote process protects its window
          * attach/detach operations with an acc-lock */
         ret = ompi_osc_state_lock(module, target, &lock_acquired, true);
-        if (ret != OMPI_SUCCESS) {
+        if (OMPI_SUCCESS != ret) {
             return ret;
         }
     }
 
+    temp_buf = calloc(remote_state_len, 1);
```

Comment:
I don't think this needs to be a `calloc`, a `malloc` should be sufficient. Mind changing that?

Reply:
Decided to keep the calloc after an offline discussion - not being 100% about performance benefit and implications with the following UCX calls.